lib/rules/no-multi-spaces.js

```diff
@@ -93,7 +93,7 @@ module.exports = {
                 const sourceCode = context.getSourceCode(),
                     source = sourceCode.getText(),
                     allComments = sourceCode.getAllComments(),
-                    pattern = /[^\n\r\u2028\u2029\t ].? {2,}/g;  // note: repeating space
+                    pattern = /[^\s].? {2,}/g;  // note: repeating space
```

Comment:
Actually, looking at the reference, `\s` is the same as [\f\n\r\t\v\u00A0\u2028\u2029], so it looks like you still picked up a few extra constrains. Not sure if it matters.